Simulator/Connections/NG911/Connections911.h

```diff
@@ -73,6 +72,14 @@ class Connections911 : public Connections {
    ///  @return true if successful, false otherwise.
    virtual bool updateConnections(AllVertices &vertices) override;
 
+   /// Finds the outgoing edge from the given vertex to the Responder closest to
+   /// the emergency call location
+   ///
+   /// @param call         The call that needs a Responder
+   /// @param vertexIdx    The index of the vertex serving the call (A PSAP)
+   /// @return    The index of the outgoing edge to the closest Responder
+   BGSIZE getEdgeToClosestResponder(const Call &call, const BGSIZE vertexIdx);
```

Comment:
Should this be `private`?

Reply:
This method is used by `All911Vertices::advanceVertices` to dispatch the appropriate responder. I decided to put it in `Connections911` because that is the class that owns the edges and edgeIndexMap. The other option was to place it in `All911Vertices` passing references to the edges and edgeIndexMap along with the call and vertex index.

Reply:
I gave this some thought and this is still what makes the most sense to me. And in this case, it needs to be a public method.

Reply:
OK, that's a good reason.